members/members/danvgar.ts

```diff
@@ -0,0 +1,55 @@
+import type { MemberObject } from '../types';
+import { profileMasks } from '../flare';
+
+// Change _EXAMPLE to `yourGitHubUserName` and add your info below
+export const _EXAMPLE: MemberObject = {
```

Comment:
This should be updated to match your username:

```suggestion
export const danvgar: MemberObject = {
```

Reply:
Oo, thanks for catching that @danieltott ! Updated in the last commit, let me know if you notice anything else.